joblib/test/test_numpy_pickle.py

```diff
@@ -1054,3 +1084,55 @@ def test_lz4_compression_without_lz4(tmpdir):
     with raises(ValueError) as excinfo:
         numpy_pickle.dump(data, fname + '.lz4')
     excinfo.match(msg)
+
+
+@with_numpy
+@parametrize('protocol', range(0, pickle.HIGHEST_PROTOCOL + 1))
```

Comment:
I don't know what to do about this parametrisation.

Historically I think that different protocols were tested because there was a problem with protocol 4 that introduced frames in #570 for example https://github.com/joblib/joblib/pull/570#issuecomment-344946296. I don't think it is worth testing all the protocols here so maybe only test the default protocol? or a few of them 3 to HIGHEST_PROTOCOL?

Reply:
done